packages/components/ContextualMenu/src/ContextualMenu.tsx

```diff
@@ -34,6 +34,21 @@ export const ContextualMenu = compose<ContextualMenuType>({
   usePrepareProps: (userProps: ContextualMenuProps, useStyling: IUseComposeStyling<ContextualMenuType>) => {
     const { setShowMenu, maxHeight, maxWidth, shouldFocusOnMount = true, shouldFocusOnContainer = false, ...rest } = userProps;
 
+    /**
+     * On macOS, focus isn't placed by default on the first focusable element. We get around this by focusing on the inner FocusZone
```

Comment:
Interesting that the OS behavior for menus actually isn't to put focus in the first item of the root menu. This fix is an awesome improvement over the current state but ideally wouldn't we want the root menu to only get focus on the first key up/down press?

May be worth noting that OS difference if we do end up not aligning with NSMenus there.

Reply:
Definitely makes sense for submenus though as that does mimic NSMenu submenu keyboard navigation/accessibility.

Reply:
Yes, @mischreiber brought up the same point to me as well. I have an idea of how we might emulate that behavior (Have a custom container view for the menu with overrides for ArrowUp and ArrowDown, and focus on that instead). 

I _may_ file that as a separate bug under "polish", since this specific big has been lurking for a while. I can update the comment here though!

Reply:
I went with filing a bug in ADO and not updating the comment because I couldn't think of a way to word it without getting _real_ specific (in a way that wouldn't help say, a new windows developer looking at this who doesn't know what an NSMenu is)